src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Quadrangle {
+
+    public Square(String color, int width, int height) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for width and height and pass them on for better code readability. Also, for a square, the width and height should be the same, so only one parameter is needed.